src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/GetApprenticeshipsRequestMapper.cs

```diff
@@ -13,12 +13,12 @@ namespace SFA.DAS.ProviderCommitments.Web.Mappers
     public class GetApprenticeshipsRequestMapper : IMapper<GetApprenticeshipsRequest, ManageApprenticesViewModel>
     {
         private readonly ICommitmentsApiClient _client;
-        private readonly IMapper<GetApprenticeshipsResponse.ApprenticeshipDetailsResponse, ApprenticeshipDetailsViewModel> _mapper;
+        private readonly IModelMapper _modelMapper;
 
-        public GetApprenticeshipsRequestMapper(ICommitmentsApiClient client, IMapper<GetApprenticeshipsResponse.ApprenticeshipDetailsResponse, ApprenticeshipDetailsViewModel> mapper)
+        public GetApprenticeshipsRequestMapper(ICommitmentsApiClient client, IModelMapper modelMapper)
```

Comment:
As other naming comments, the request and viewmodel should be `IndexRequest` and `IndexViewModel`, and we name our mappers after the _target_ type. So this would be `IndexViewModelMapper`.

Reply:
Moved